src/components/Summary/AlterationSummary.vue

```diff
@@ -85,20 +85,60 @@
         </v-layout>
       </div>
     </template>
+
+    <!-- TODO: Name Translation -->
+
+    <!-- TODO: Pre-existing Company Provisions -->
+
+    <!-- TODO: Share Structure -->
+
+    <!-- Alteration Date and Time -->
+    <div class="ma-6 pb-6">
+      <v-container class="alteration-date-time" :class="{ 'invalid': alterationDateTimeInvalid }">
+        <v-row no-gutters>
+          <v-col cols="3" class="pr-4">
+            <label><strong>Alteration Date and Time</strong></label>
+          </v-col>
+
+          <v-col cols="9" class="pl-4">
```

Comment:
What seemed to work well when playing around with Name Request mobile was leveraging these `cols`.
By defining the `cols=12 md=3 lg=3`or something along those lines would prevent this two cols from being compressed beside each other on mobile.

Obviously no changes required here, just sharing my thoughts on what the majority of the issue was in Name Request.